SCAutolib/controller.py

```diff
@@ -161,7 +160,7 @@ def setup_system(self, install_missing: bool, gdm: bool, graphical: bool):
             packages += ["pcsc-lite-ccid", "pcsc-lite", "virt_cacard",
                          "vpcd", "softhsm"]
             extra_args = ""
-            if os_version in (OSVersion.RHEL_10, OSVersion.CentOS_10):
+            if isDistro(['rhel', 'centos'], version='10'):
```

Comment:
should we make here some logic to avoid the need to repeat `['rhel', 'centos']` to get `rhel` based distros? Do we have some different code path for rhel and centos somewhere?

Reply:
Well, we do not have any different path somewhere for RHEL and CentOS but I am not sure about the future so I am more inclined to keep it like that. We can implement quickly a special flag like "rhel-based" that would have both but does it make that much of the difference or is it only a stylistic thing?

Reply:
Sure. Lets keep it like this. If this would be an issue in the future, lets change it in the future